### PR TITLE
docs: add cadence update post

### PR DESCRIPTION
**Pull request by codebytere**
Address upcoming changes to Electron's Release schedule.

cc @electron/wg-releases

### CODE REVIEW

File: data/blog/cadence-pause.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Upcoming Electron Releases
+author: codebytere
+date: '2019-05-13'
```
Comment (sofianguy) [inline data/blog/cadence-pause.md]: wrong date ⏰

Comment (sofianguy) [reply data/blog/cadence-pause.md]: we'd run into that issue where the blog wasn't showing up on the website, when we published E5's blog (if you remember that @codebytere ) 😂

File: data/blog/cadence-pause.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## What's Happening?
+
+Our major release cadence schedule moves in lockstep with that of Chromium, and the Chromium project has made the recent decision to [pause its releases](https://blog.chromium.org/2020/03/upcoming-chrome-releases.html) due to adjusted work schedules. This means that for the duration of Chromium's altered cadence, Electron will also temporarily pause new major releases.
+
+We feel that our best choice is to follow in Chromium's footsteps, and so in the interim the Electron team will shift to full-time work on bugfixes, security, performance, and stability.
+
+We want to ensure that both our maintainers and our consumers' wellbeing is prioritized during this time, so we welcome your feedback and look forward to returning to our regular release schedule.
+
+For upcoming updates, please follow our [twitter account](https://twitter.com/electronjs).
```
Comment (malept) [inline data/blog/cadence-pause.md]:
```suggestion
For more updates, please follow our [Twitter account](https://twitter.com/electronjs).
```

File: data/blog/cadence-pause.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+date: '2020-03-19'
+---
+
+Electron is temporarily pausing major releases
+
+---
+
+## What's Happening?
+
+Our major release cadence schedule moves in lockstep with that of Chromium, and the Chromium project has made the recent decision to [pause its releases](https://blog.chromium.org/2020/03/upcoming-chrome-releases.html) due to adjusted work schedules. This means that for the duration of Chromium's altered cadence, Electron will also temporarily pause new major releases.
```
Comment (malept) [inline data/blog/cadence-pause.md]: I think this link structure works?

```suggestion
Our [major release cadence schedule](12-week-cadence) moves in lockstep with that of Chromium, and the Chromium project has made the recent decision to [pause its releases](https://blog.chromium.org/2020/03/upcoming-chrome-releases.html) due to adjusted work schedules. This means that for the duration of Chromium's altered cadence, Electron will also temporarily pause new major releases.
```

Comment (codebytere) [reply data/blog/cadence-pause.md]: i can just link to the actual post

**Pull request merged**